generators/app/templates/src/services/AuthServices.ts

```diff
@@ -0,0 +1,37 @@
+import { ApiResponse } from 'apisauce';
+
+import { User, Credentials } from '~app/contexts/UserContext/reducer';
+
+import api from '../config/api';
+
+import LocalStorageService from './LocalStorageService';
+
+const TOKEN_FIELD_NAME = 'sessionToken';
+
+export interface LoginError {
+  message: string;
+}
+
+export const setCurrentUser = (currentUser: User) => {
+  api.setHeader('Authorization', currentUser.sessionToken);
```

Comment:
This is fine I guess, but you're not removing the header when you remove user.

Reply:
will add the counterpart on logout